seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -241,10 +360,70 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_f
         # Test saving ID file exception
         url = reverse(create_project_from_workspace, args=[TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME2])
         mock_mv_file.side_effect = Exception('Something wrong while moving the ID file.')
+        # Test triggering dag exception
+        responses.replace(responses.PATCH, 
+                      '{}/api/v1/variables/AnVIL_WES'.format(MOCK_AIRFLOW_URL), 
+                      json={'test': 2},
```

Comment:
what does this body do? if its a generic error response it is more readable to do something like `{'error': 'not found'}` or anything thats more clearly an error